src/sas/sascalc/fit/BumpsFitting.py

```diff
@@ -353,9 +355,12 @@ def abort_test():
     try:
         best, fbest = fitdriver.fit()
         errors = []
+        stack_trace = None
     except Exception as exc:
         best, fbest = None, np.NaN
-        errors = [str(exc), traceback.format_exc()]
+        exc_type, exc_value, exc_tb = sys.exc_info()
```

Comment:
why use `sys.exc_info` when you already have all the information in `traceback.format_exc`?

Reply:
`traceback.format_exc()` returns a string, whereas sys.exc_info() returns a traceback object, so instead of parsing a string, the traceback methods and attributes can be used in order to get information on failing code.